### ISSUE TITLE
Pre-trained biases are ignored when batch_norm is present using tf-slim API

### ISSUE THREAD

**Issue by sjain-stanford** [jinyu121/DW2TF#3]
### Issue
As per tf-slim [conv2d documentation](https://www.tensorflow.org/api_docs/python/tf/contrib/layers/conv2d):

> normalizer_fn: Normalization function to use instead of biases. If normalizer_fn is provided then biases_initializer and biases_regularizer are ignored and biases are not created nor added. default set to None for no normalizer function

In [DW2TF](https://github.com/jinyu121/DW2TF/blob/9db5b2c840c7f0bc06befe63f06c8d248ef8e234/util/cfg_layer.py#L47-L66), biases were being assigned using `biases_initializer`, which would silently get ignored when `normalizer_fn` was provided. 

### Examples
**Conv without BN (biases are populated correctly)**
![without_bn](https://user-images.githubusercontent.com/19234106/51418995-f8a39000-1b3b-11e9-8b9f-fc86a559460e.PNG)

**Conv with BN (biases are ignored)**
![with_bn](https://user-images.githubusercontent.com/19234106/51419002-fd684400-1b3b-11e9-9624-103e0dacc127.PNG)

### Solution
The correct way to handle this is to use the `beta` initializer of the normalizer for biases.

I've fixed this using tf.layers, where biases are added to convolution when BN is absent, or absorbed in `beta` of BN when present. PR is on the way.
